components/filter-components/Timeline.vue

```diff
@@ -23,6 +23,7 @@
 
 <script>
 import Slider from '@vueform/slider'
+import '@vueform/slider/themes/default.css'
```

Comment:
I'm not certain but I don't think the slider is having any effect on the witches on the map. They aren't changing for me. Thanks Ruby. This might be expected and dealt with in another PR but wanted to flag here in case that isn't the case.

Reply:
I still don't think this is working @rubyimrie  - if you fix again, could you detail the fix and issue and leave it unresolved? Thank you

Reply:
if (this.clustersOn) {
        this.clustersOn = !this.clustersOn;
      }
      
      have fixed, this bit of code i had commented out when the cluster map wasn't working and had forgotten to   change it back, when the timeline slider appears the map changes to normal map so that the changes are visible because with the clustering on the changes aren't noticeable

Reply:
That's it working now. Thank you!